Makefile

```diff
@@ -0,0 +1,26 @@
+all: build
+
+FLAGS =
+ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+REGISTRY = gcr.io/google_containers
+TAG = v0.2.0
+
+deps:
+	go get github.com/tools/godep
+
+build: clean deps
+	$(ENVVAR) godep go build -o kube-state-metrics 
+
+test-unit: clean deps build
+	$(ENVVAR) godep go test --race . $(FLAGS)
+
+container: build
```

Comment:
The file was largely taken from the rescheduler repository. Figured we just keep it consistent.